eng/tools/sdk-suppressions/src/index.ts

```diff
@@ -0,0 +1,31 @@
+
+import { exit } from "process";
+import { updateSdkSuppressionsLabels, PullRequestContext } from "./updateSdkSuppressionsLabel.js";
+
+function getUsage(): string {
+    return (
+      "Parameter missing! \n" + 
+      "Check GITHUB_PULL_REQUEST_CONTEXT where defined in step: Get GitHub PullRequest Context" +
+      "Check OUTPUT_FILE in step env: Run get suppressions label script " +
```

Comment:
Tools should write output to the console, not to a file specified via an env var.

Reply:
change error message